modin/core/execution/dask/implementations/pandas_on_dask/partitioning/partition.py

```diff
@@ -317,31 +311,36 @@ def apply_func(partition, func, *args, **kwargs):
     partition : pandas.DataFrame
         A pandas DataFrame the function needs to be executed on.
     func : callable
-        Function that needs to be executed on `partition`.
-    *args
-        Additional positional arguments to be passed in `func`.
-    **kwargs
-        Additional keyword arguments to be passed in `func`.
+        The function to perform.
+    *args : list
+        Positional arguments to pass to ``func``.
+    **kwargs : dict
+        Keyword arguments to pass to ``func``.
 
     Returns
     -------
     pandas.DataFrame
         The resulting pandas DataFrame.
     str
         The node IP address of the worker process.
+
+    Notes
+    -----
+    Directly passing a call queue entry (i.e. a list of [func, args, kwargs]) instead of
+    destructuring it causes a performance penalty.
```

Comment:
This is applicable for Ray too so let's add the changes for Ray engine as well.

Reply:
Added this comment to ray's _apply_func.